src/Microsoft.DotNet.Darc/src/Darc/Program.cs

```diff
@@ -86,5 +169,105 @@ private static Type[] GetOptions()
                     typeof(GetGoalCommandLineOptions)
                 };
         }
+
+        private static void InitializeTelemetry()
+        {
+            var isDebugging = Debugger.IsAttached;
+            var config = TelemetryConfiguration.CreateDefault();
+            if (!isDebugging)
+            {
+                config.InstrumentationKey = "9fcef3c7-f401-41c7-9e91-1f6029c8dcc3";
```

Comment:
Should we stick this GUID in a config file and pull it in with DI options?

Reply:
We could, but that means we need to make sure the config file makes it through all the layers of moving the exe around... and also hook up the entire config path just for one value. :-)

If we already _had_ a config story, I'd say we'd put this there, but hooking all that up seems overkill for this one value. :-)

Reply:
For comparison, here is the usage that "dotnet.exe" has. :-) https://github.com/dotnet/sdk/blob/72d3792bb1e5ee56841408009a8638d51acd948b/src/Cli/dotnet/Telemetry/Telemetry.cs#L24